Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Startup.cs

```diff
@@ -32,25 +39,28 @@ public void ConfigureServices(IServiceCollection services)
 
             var connection = @"Server=dev.retrotest.co.za;Database=ereader;User Id=group4;Password=3bHNuE8&rvG+99U2;";
```

Comment:
Should there be space between *User Id* = group4?

Reply:
Hi ash, the connection string has been tested and it is currently working. (I'm on front end generally so just picking up from where they left off) as far as I understand apparently yes?

Reply:
I'm happy